projects/material-addons/src/themes/common/styles.scss

```diff
@@ -21,42 +25,42 @@ $contrast: (
   A700: theming.$dark-primary-text,
 );
 
-@function get-main-color($color-scheme) {
-  @return map-get($color-scheme, 500);
-}
-
-@function get-selection-background($color-scheme) {
-  @return map-get($color-scheme, 50);
-}
-
-@function get-table-hover-color($color) {
-  @return rgba($color, 0.08);
-}
-
-@function get-background-color($color-scheme) {
-  @return map-get($color-scheme, background-color);
-}
-
-@mixin build-custom-theme($primary-palette, $warn-palette: mat.$red-palette, $theme-name: 'undefined') {
-  $material-addons-colors-primary: mat.define-palette($primary-palette);
-  $material-addons-colors-warn: mat.define-palette($warn-palette);
-  //$material-addons-theme: mat.define-light-theme($material-addons-colors-primary, $material-addons-colors-primary, $material-addons-colors-warn);
-
-  $material-addons-theme: mat.define-light-theme(
-    (
-      color: (
-        primary: $material-addons-colors-primary,
-        accent: $material-addons-colors-primary,
-        warn: $material-addons-colors-warn,
-      ),
-    )
-  );
-
-  $hover-color: mat.define-palette($primary-palette, 50);
+$default-palette: (
+  panel-background-color: #e8e8e8,
+  panel-border-color: #cccccc,
+  panel-select-background: white,
+);
 
-  @include mat.all-component-themes($material-addons-theme);
-  //setting the density for the components
-  //@include mat.form-field-density(-5);
+@mixin color($theme-name, $theme) {
+  $main-primary: theme.get-main-color();
+  $selection-background: theme.get-selection-background();
+  $table-hover-color: theme.get-table-hover-color($main-primary);
+  $background-color: theme.get-background-color();
+  $warn-color: map-get(theme.$primary-palette, warn-color);
+  $hover-color: map-get(theme.$primary-palette, 50);
+  $panel-color: map-get(theme.$primary-palette, 900);
+  $panel-background-color: theme.map-get-or-default(theme.$primary-palette, panel-background-color);
+  $panel-border-color: theme.map-get-or-default(theme.$primary-palette, panel-border-color);
+  $panel-select-background: theme.map-get-or-default(theme.$primary-palette, panel-select-background);
+
+  // Remark: when we style the addons components there is no theme set, yet. So we have multiple possibilities:
```

Comment:
Maybe this would be some good comment for a Readme?

Reply:
Not sure where to place it in the Addons. This way everybody will see it